# videodev.h is not anymore located in the Linux package since 2.6.38, it has
# been moved and renamed.
# Closes: #621968

diff -Naur fotowall-0.9.orig/3rdparty/videocapture/VideoDevice.h fotowall-0.9/3rdparty/videocapture/VideoDevice.h
--- fotowall-0.9.orig/3rdparty/videocapture/VideoDevice.h	2009-11-29 13:59:59.000000000 +0100
+++ fotowall-0.9/3rdparty/videocapture/VideoDevice.h	2011-04-12 20:47:49.000000000 +0200
@@ -41,7 +41,7 @@
     #endif
     #include <linux/fs.h>
     #include <linux/kernel.h>
-    #include <linux/videodev.h>
+    #include <libv4l1-videodev.h>
     #ifdef V4L2_CAP_VIDEO_CAPTURE
         #define VD_BUILD_LINUX_TWO              ///< build V4L2 code too
     #endif
